snippets/jsim.json

```
@@ -29,11 +29,60 @@
 		],
 		"description": "Code snippet for an include statement"
 	},
-	"Custom component": {
+	"User-defined component": {
 		"prefix": "x",
 		"body": [
 			"X${1:name} ${2:inputs} ${3:outputs} ${4:gate}"
 		],
-		"description": "Code snippet for a custom component"
+		"description": "Code snippet for a user-defined component"
+	},
+	"Resistor": {
+		"prefix": "r",
+		"body": [
+			"R${1:name} ${2:inputs} ${3:resistance}"
+		],
+		"description": "Code snippet for a resistor"
+	},
+	"Capacitor": {
+		"prefix": "c",
+		"body": [
+			"C${1:name} ${2:inputs} ${3:capacitance}"
+		],
+		"description": "Code snippet for a capacitor"
+	},
+	"Inductor": {
+		"prefix": "l",
+		"body": [
+			"L${1:name} ${2:inputs} ${3:inductance}"
+		],
+		"description": "Code snippet for a capacitor"
```

I think you forgot to change this description for the inductor:

```suggestion
		"description": "Code snippet for an inductor"
```

---

Oops, this is why you should never copy paste code :)